ckanext/canada/templates/error_document_template.html

```diff
@@ -38,6 +43,15 @@ <h1>{{ code }} {{ name }}</h1>
           <p class="mrgn-tp-lg">{{ _('We encountered an error') }}</p>
         {% endif %}
       {% endif %}
+      {% if g.debug %}
```

Comment:
in debug mode we get the werkzeug debugger not this page right?

Reply:
I think that is only if you have the dev-requirements installed. Otherwise it outputs like a small accordion section. I think we can just safely remove this from our templates, we should not even output it in prod servers. I think we can all read server logs.